src/Vatras/Lang/All.agda

```diff
@@ -12,22 +12,57 @@ import Vatras.Lang.2CC
 import Vatras.Lang.NADT
 import Vatras.Lang.ADT
 import Vatras.Lang.OC
+import Vatras.Lang.FST
 import Vatras.Lang.Gruler
 
+open import Data.Empty.Polymorphic using (⊥)
+open import Vatras.Util.Nat.AtLeast using (ℕ≥)
+open import Size using (∞)
+open import Vatras.Framework.Definitions using (𝔽; 𝔸; 𝕍)
+open import Vatras.Framework.Variants using (Rose)
+
 module VariantList = Vatras.Lang.VariantList
-module CCC         = Vatras.Lang.CCC
-module NCC         = Vatras.Lang.NCC
-module 2CC         = Vatras.Lang.2CC
-module NADT        = Vatras.Lang.NADT
-module ADT         = Vatras.Lang.ADT
-module OC          = Vatras.Lang.OC
-module Gruler      = Vatras.Lang.Gruler
 
-module FST where
-  open import Size using (∞)
-  open import Vatras.Framework.Definitions using (𝔽; 𝔸)
-  open import Vatras.Framework.Variants using (Rose)
-  open import Vatras.Lang.FST hiding (⟦_⟧) public
+module CCC where
+  open Vatras.Lang.CCC using (CCC; CCCL; Configuration) public
+  module _ {F : 𝔽} where
+    open Vatras.Lang.CCC F hiding (CCC; CCCL; Configuration) public
```

Comment:
I would be great if you could document (once at the top of this file?) why there are so many modules. I guess you are just turning explicit into implicit parameters for easier usage? How does this help exactly?

Reply:
EDIT: I see now that you change explicit to implicit parameters just for some of the definitions: Not for languages and configurations but for everything else (including semantics).